diff --git a/pokemonred_puffer/__pycache__/environment.cpython-310.pyc b/pokemonred_puffer/__pycache__/environment.cpython-310.pyc
index e2dad13..ba1e01f 100644
Binary files a/pokemonred_puffer/__pycache__/environment.cpython-310.pyc and b/pokemonred_puffer/__pycache__/environment.cpython-310.pyc differ
diff --git a/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc b/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc
index 69c24c9..41f1002 100644
Binary files a/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc and b/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc differ
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index a64813c..5a19e4a 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20260416_114611-8mx4trq1/logs/debug-internal.log
\ No newline at end of file
+run-20260418_152653-la9htkp4/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index dcec80c..95eb924 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20260416_114611-8mx4trq1/logs/debug.log
\ No newline at end of file
+run-20260418_152653-la9htkp4/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 5ea392b..50bda58 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20260416_114611-8mx4trq1
\ No newline at end of file
+run-20260418_152653-la9htkp4
\ No newline at end of file
diff --git a/wandb/wandb-resume.json b/wandb/wandb-resume.json
index 09f7848..f92b8e9 100644
--- a/wandb/wandb-resume.json
+++ b/wandb/wandb-resume.json
@@ -1 +1 @@
-{"run_id": "8mx4trq1"}
\ No newline at end of file
+{"run_id": "la9htkp4"}
\ No newline at end of file
