diff --git a/pokemonred_puffer/__pycache__/cleanrl_puffer.cpython-310.pyc b/pokemonred_puffer/__pycache__/cleanrl_puffer.cpython-310.pyc
index aa52873..61ac9f6 100644
Binary files a/pokemonred_puffer/__pycache__/cleanrl_puffer.cpython-310.pyc and b/pokemonred_puffer/__pycache__/cleanrl_puffer.cpython-310.pyc differ
diff --git a/pokemonred_puffer/__pycache__/environment.cpython-310.pyc b/pokemonred_puffer/__pycache__/environment.cpython-310.pyc
index ba1e01f..48c5c0a 100644
Binary files a/pokemonred_puffer/__pycache__/environment.cpython-310.pyc and b/pokemonred_puffer/__pycache__/environment.cpython-310.pyc differ
diff --git a/pokemonred_puffer/__pycache__/eval.cpython-310.pyc b/pokemonred_puffer/__pycache__/eval.cpython-310.pyc
index 1b06de5..ec08e12 100644
Binary files a/pokemonred_puffer/__pycache__/eval.cpython-310.pyc and b/pokemonred_puffer/__pycache__/eval.cpython-310.pyc differ
diff --git a/pokemonred_puffer/__pycache__/global_map.cpython-310.pyc b/pokemonred_puffer/__pycache__/global_map.cpython-310.pyc
index 3fed4de..5848b72 100644
Binary files a/pokemonred_puffer/__pycache__/global_map.cpython-310.pyc and b/pokemonred_puffer/__pycache__/global_map.cpython-310.pyc differ
diff --git a/pokemonred_puffer/__pycache__/train.cpython-310.pyc b/pokemonred_puffer/__pycache__/train.cpython-310.pyc
index 28be1f0..ee4e35d 100644
Binary files a/pokemonred_puffer/__pycache__/train.cpython-310.pyc and b/pokemonred_puffer/__pycache__/train.cpython-310.pyc differ
diff --git a/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc b/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc
index 41f1002..2679979 100644
Binary files a/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc and b/pokemonred_puffer/rewards/__pycache__/baseline.cpython-310.pyc differ
diff --git a/pokemonred_puffer/wrappers/__pycache__/exploration.cpython-310.pyc b/pokemonred_puffer/wrappers/__pycache__/exploration.cpython-310.pyc
index 73eea6d..d1c6f52 100644
Binary files a/pokemonred_puffer/wrappers/__pycache__/exploration.cpython-310.pyc and b/pokemonred_puffer/wrappers/__pycache__/exploration.cpython-310.pyc differ
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 4a4b1e2..7399801 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20260418_155547-7g4a12co/logs/debug-internal.log
\ No newline at end of file
+run-20260420_143552-4731qhq4/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index aa79240..e9c5170 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20260418_155547-7g4a12co/logs/debug.log
\ No newline at end of file
+run-20260420_143552-4731qhq4/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 9bf4cd8..c67c201 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20260418_155547-7g4a12co
\ No newline at end of file
+run-20260420_143552-4731qhq4
\ No newline at end of file
diff --git a/wandb/wandb-resume.json b/wandb/wandb-resume.json
index 3a2917f..a82db62 100644
--- a/wandb/wandb-resume.json
+++ b/wandb/wandb-resume.json
@@ -1 +1 @@
-{"run_id": "7g4a12co"}
\ No newline at end of file
+{"run_id": "4731qhq4"}
\ No newline at end of file
